curriculum/challenges/english/25-front-end-development/lab-poker-hand-page/66be24cb4144f955b6bcc550.md

```diff
@@ -0,0 +1,320 @@
+assert.isAtLeast(els.length, 1);
+els.forEach(el => {
+  const children = el.children;
+  assert.lengthOf(children, 3);
+  Array.from(children).forEach(child => {
+    assert.equal(child.tagName, 'DIV');
+  });
+});
+```
+
+You should have a `div` with a class of `left` within each `.card` element.
+
+```js
+const els = document.querySelectorAll('.card');
+assert.isAtLeast(els.length, 1);
+els.forEach(el => assert.exists(el.querySelector('div.left')));
+```
+
+You should have a `div` with a class of `middle` within each `.card` element.
+
+```js
+const els = document.querySelectorAll('.card');
+assert.isAtLeast(els.length, 1);
+els.forEach(el => assert.exists(el.querySelector('div.middle')));
+```
+
+You should have a `div` with a class of `right` within each `.card` element.
+
+```js
+const els = document.querySelectorAll('.card');
+assert.isAtLeast(els.length, 1);
+els.forEach(el => assert.exists(el.querySelector('div.right')));
+```
+
+You should have a `#poker-hand` selector that sets its elements' `display` to `flex`.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('#poker-hand')?.display, 'flex');
+```
+
+You should have a `#poker-hand` selector that sets its elements' `justify-content` to `center`.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('#poker-hand')?.justifyContent, 'center');
+```
+
+You should have a `#poker-hand` selector that sets its elements' `flex-wrap` to `wrap`.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('#poker-hand')?.flexWrap, 'wrap');
+```
+
+You should have a `#poker-hand` selector that sets its elements' `gap` to `20px`.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('#poker-hand')?.gap, '20px');
+```
+
+You should have a `.card` selector that sets its elements' `display` to `flex`.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('.card')?.display, 'flex');
+```
+
+You should have a `.card` selector that sets its elements' `justify-content` to `space-between`.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('.card')?.justifyContent, 'space-between');
+```
+
+You should have a `.left` selector that sets its elements' `align-self` to `flex-start`.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('.left')?.alignSelf, 'flex-start');
+```
+
+You should have a `.middle` selector that sets its elements' `align-self` to `center`.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('.middle')?.alignSelf, 'center');
+```
+
+You should have a `.right` selector that sets its elements' `align-self` to `flex-end`.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('.right')?.alignSelf, 'flex-end');
+```
+
+You should have a `.middle` selector that sets its elements' `flex-direction` to `column`.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('.middle')?.flexDirection, 'column');
+```
+
+# --seed--
+
+## --seed-contents--
+
+```html
+
```

Comment:
Aren't we going to provide the boilerplate at this point?

Reply:
I'm not entirely sure. Do we have any standard for that? Like all the CSS projects have the basic HTML boilerplate?

Reply:
I don't think we have a standard but a couple of labs that have been merged have the boilerplate as seed code.